src/main/java/frc/robot/Robot.java

```diff
@@ -78,10 +153,30 @@ public void autonomousInit() {
 
   /**
    * This function is called periodically during autonomous.
+ * @return 
    */
+  double distanceToWall;
   @Override
   public void autonomousPeriodic() {
-  }
+    double angleCos;
```

Comment:
add docs on purpose of this
Thanks!

(i'm reading it as a constant update to the angle and distance variables, but make that explicitly obvious at the top of the block so other people know exactly what this does)